templates/service.yaml

```diff
@@ -4,6 +4,10 @@ metadata:
   name: {{ include "ld-relay.fullname" . }}
   labels:
     {{- include "ld-relay.labels" . | nindent 4 }}
+  {{- with .Values.service.annotations }}
```

Comment:
Saying it back for understanding. This basically would add this section if `.Values.service.annotations has a value`?